.jenkins/github/platform-management-sdk-tests.Jenkinsfile

```diff
@@ -0,0 +1,190 @@
+                        }
+                        echo "Running from branch ${env.ghprbSourceBranch} on ${INSTANCE_NAME}.stage.c8y.io"
+                    }
+                }
+            }
+        }
+
+        stage('Get platform version') {
+            environment {
+                ADMIN_CREDENTIALS = credentials("${ADMIN_CREDENTIALS_ID}")
+            }
+            steps {
+                sh 'curl -k -u ${ADMIN_CREDENTIALS} http://management.'+ INSTANCE_NAME + '.stage.c8y.io:8111/tenant/system/options/system/version > system-version.json'
+                script {
+                    def systemVersion = readJSON file: 'system-version.json'
+                    env.SYSTEM_VERSION = systemVersion.value
+                }
+                echo "Running tests for platform version ${env.SYSTEM_VERSION} on ${INSTANCE_NAME}.stage.c8y.io"
+            }
+        }
+
+        stage('Checkout clients-java repository') {
+            steps {
+                checkoutRepo('cumulocity-clients-java', "refs/tags/clients-java-${env.SYSTEM_VERSION}")
+            }
+        }
+
+        stage('Execute SDK tests') {
+            environment {
+                ADMIN_CREDENTIALS = credentials("${ADMIN_CREDENTIALS_ID}")
+                MVN_SETTINGS = credentials("maven-settings")
+            }
+            steps {
+                container('java') {
+                    script {
+                        catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
+                            sh """
+                            cd cumulocity-clients-java
+                            ./mvnw -B -s $MVN_SETTINGS verify \
+                              -Pintegration -Dcumulocity.host=http://${INSTANCE_NAME}.stage.c8y.io \
+                              -Dcumulocity.management.password=$ADMIN_CREDENTIALS_PSW \
+                              -Dcumulocity.management.username=$ADMIN_CREDENTIALS_USR
+                            """
+                        }
+                    }
+                }
+            }
+            post {
+                always {
+                    junit(testResults: '**/TEST-*.xml', keepLongStdio: true, allowEmptyResults: true)
+                    archiveArtifacts(artifacts: '**/TEST-*.xml', allowEmptyArchive: true)
+                }
+                success {
+                    createSummary(icon: 'green.gif', text: 'SDK tests passed')
+                }
+                failure {
+                    createSummary(icon: 'error.gif', text: 'SDK tests failed')
+                }
+            }
+        }
+    }
+    post {
+        always {
+            script {
+                if (env.ghprbSourceBranch.contains('/Staging')) {
+                    build job: "cucumber-tenants-cleanup",
+                        parameters: [
+                                string(name: "DOMAIN", value: "${INSTANCE_NAME}.stage.c8y.io"),
+                                string(name: "TENANT_PREFIX", value: "sdk_tenant"),
```

Comment:
I'd leave some team indicator in the tenant prefix, i.e. `plat-mgmt-sdk`

Reply:
sure